metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/_config.py

```diff
@@ -68,11 +71,19 @@ class DatahubLineageConfig(ConfigModel):
 
     disable_openlineage_plugin: bool = True
 
-    def make_emitter_hook(self) -> "DatahubGenericHook":
+    def make_emitter_hook(self) -> Union["DatahubGenericHook", "DatahubCompositeHook"]:
         # This is necessary to avoid issues with circular imports.
-        from datahub_airflow_plugin.hooks.datahub import DatahubGenericHook
-
-        return DatahubGenericHook(self.datahub_conn_id)
+        from datahub_airflow_plugin.hooks.datahub import (
+            DatahubCompositeHook,
+            DatahubGenericHook,
+        )
+
+        conn_ids = self.datahub_conn_id.split(",")
```

Comment:
imo this splitting should happen in a pydantic validator on the DatahubLineageConfig